src/gobprepare/cloner/oracle_to_postgres.py

```diff
@@ -185,33 +240,54 @@ def _copy_table_data(self, table_definition: Tuple[str, List]) -> int:
         outer_select = self._get_select_list_for_table_definition(table_definition)
         inner_select = ','.join([column_name for column_name, column_type in column_definitions])
 
-        query = f"SELECT /*+ PARALLEL */ {outer_select} FROM (SELECT {inner_select} FROM {full_table_name})"
-        cursor = query_oracle(self._src_connection, [query])
-
+        ids = self._get_ids_for_table(full_table_name)
         row_cnt = 0
-        while True:
+
+        chunks = self._list_to_chunks(ids, self.READ_BATCH_SIZE)
+        order_field = self._get_id_columns_for_table(full_table_name)[0]
+
+        for chunk in chunks:
+            if chunk['min'] or chunk['max']:
+                where_max = f"{order_field} < '{chunk['max']}'" if chunk['max'] is not None else ""
+                where_min = f"{order_field} >= '{chunk['min']}'" if chunk['min'] is not None else ""
+                where = f"WHERE {where_min} {'AND' if where_min and where_max else ''} {where_max}"
```

Comment:
I find the number of None clauses a little confusing, but...
I tried for a long time to convert this in an 'AND '.join([...]) but didn't manage to produce any 'better' code.